app-android-xamarin/Moodis/Moodis/Resources/layout/CameraFragmentLayout.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rename this file to `fragment_camera.xml`, so it would be in same naming format as activity layout files